docs/user/profiling.rst

```diff
@@ -0,0 +1,79 @@
+.. _profiling:
+
+Profiling
+=========
+
+In this section you can find some tips on how to profile your scala-native binary in Linux.
```

Comment:
I think this line should have _Scala Native_ instead of _scala-native_.

Reply:
OK thanks; I have fixed that now and rebased.